bescort.lic

```diff
@@ -584,6 +590,64 @@ class Bescort
       end
     end
   end
+  
+  def take_theren_rope_bridge(mode)
+	theren_side_roomid = 8650
+	rossman_side_roomid = 8637
+	@target_room = false
```

Comment:
This (and line 598) don't need to be instance vars (i.e. you can remove the `@` from the name).